flow/actorcompiler/Program.cs

```diff
@@ -33,16 +33,21 @@ public static int Main(string[] args)
             if (args.Length < 2)
             {
                 Console.WriteLine("Usage:");
-                Console.WriteLine("  actorcompiler [input] [output]");
+                Console.WriteLine("  actorcompiler <input> <output> [--disable-actor-without-wait-error]");
```

Comment:
Can we change this to `--disable-actor-without-wait-warning`?

Reply:
Eventually I think we really do want this to be a compile error, but we can't enforce this until all supported build systems build without any "actor without wait" warnings/errors. I'll change the flag to say it's a warning

Reply:
If we do end up making it an error, then it may be reasonable to get rid of the offending tests altogether and then drop this option as well.

Reply:
I now use disable-actor-without-wait-warning consistently throughout. PTAL